python/tvm/driver/tvmc/compiler.py

```diff
@@ -319,19 +336,18 @@ def compile_model(
         dump_code = [dump_code]
     dumps = {}
     for source_type in dump_code:
-        lib = graph_module.get_lib()
+        if use_vm:
+            _, lib = graph_module.save()
```

Comment:
I think we can instead do `lib = graph_module.lib()` to be a little more efficient.